guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -17,6 +17,8 @@ To use the new page by default, in Settings, set Show new host overview page to
 * Content source
 * Content view environments - change the content view and lifecycle environment of hosts
 
+*Container push* - You can now push container images to Katello via `podman push`.
```

Comment:
This is in the middle of the new All Hosts headline features. I would move it below, to just before "Assign multiple content view environments.."

Reply:
Done!